src/main/java/com/iota/iri/storage/Tangle.java

```diff
@@ -58,6 +59,9 @@ public void init() throws Exception {
         for(PersistenceProvider provider: this.persistenceProviders) {
             provider.init();
         }
+        if (cacheManager == null) {
+            cacheManager = new CacheManagerImpl(new MainnetConfig());
+        }
```

Comment:
I think this is unneccessary,
this dependency should come via DI only

Reply:
Will refactor at a later stage as agreed.